src/wiki/plugins/editsection/wiki_plugin.py

```diff
@@ -13,7 +13,7 @@ class EditSectionPlugin(BasePlugin):
     urlpatterns = {
         "article": [
             url(
-                r"^(?P<location>[0-9-]+)/header/(?P<header>\w+)/$",
+                r"^header/(?P<header>[\w-]+)/$",
```

Comment:
Will this work if two section titles are identical?

I.e.

```
# Investigation 1

## Date

# Investigation 2

## Date
```

Reply:
Sure... the TreeProcessor will ensure that each section has a unique ID (if it doesn't have one already, for example from the TOC extension). I will add a test for that to ensure.

Reply:
oh, good catch :blush:

Reply:
It sounds like a great great idea to use the same ID for the URL, this will be easy to understand and maintain :+1: